rules/tech-structures.yaml

```diff
@@ -24,3 +24,23 @@ caoild:
 		Sequence: idle-pump
 	GivesBuildableArea:
 
+cahosp:
+	Inherits: ^TechBuilding
+	Building:
+		Footprint: xxxx xxxx xxxx xxxx xxxx xxxx
+		Dimensions: 6, 4
+	ExternalCapturable:
+	ExternalCapturableBar:
```

Comment:
This and the `ExternalCapturable` above look bogus. The engineers use `Captures`.
I removed them from `caoild` in https://github.com/OpenRA/ra2/commit/81023c50732589276ebb50346e8bd7acc781a5e8 just now.